change/@fluentui-react-charting-b86a16d6-a1aa-4d4f-bb2c-b1632ca07a8d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28165&commitId=ee77ff8412ec514055e4d63644b0d73dea08a4ac&env=prod&clientType=FLUENTUI)


  <details><summary><b>Panel</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Panel.Custom anchored left.default.chromium.png|24| Changed|
|Panel.Large fixed right.default.chromium.png|24| Changed|
|Panel.Extra large right.default.chromium.png|24| Changed|
|Panel.SearchBox and Right Panel - RTL.default.chromium.png|28| Changed|
|Panel.With no header, close button.default.chromium.png|24| Changed|
|Panel.Custom.default.chromium.png|24| Changed|
|Panel.SearchBox and Right Panel.default.chromium.png|28| Changed|
|Panel.With footer at the bottom.default.chromium.png|24| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>